packages/web/src/hooks/vaults/useVaultApy.ts

```
@@ -56,7 +56,7 @@ export const useVaultApy = (vault?: IVault): IVaultApy[] => {
           vault.amountsInfo?.depositedAmount ? +vault.amountsInfo.depositedAmount.tokens : 0
         );
 
-        apys.push({ rewardController: controller, apy });
+        if (!isNaN(apy)) apys.push({ rewardController: controller, apy });
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)